### PR TITLE
fix(suite): Cursors in truncate

### LINKED ISSUE #11286: Main menu "?" cursor on account titles
**Describe the bug**
When hovering over an account title (and fiat valua) in the main menu, the cursor changes to a "?"

**Info:**

-   Suite Version: app, 24.2.1
-   OS: MacOS Sonoma

**How to reproduce**

1. Go to main menu
2. Hover on some account title or its fiat value
3. Observe cursor change to "?"

**Expected behavior**
The cursor should remain being a pointer

**Screenshots**
Feb-21-2024 17-38-44

### PR DESCRIPTION
**Pull request by jvaclavik**
## Related Issue

Resolve #11286 #11272

### CODE REVIEW

File: packages/components/src/components/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -36,11 +36,11 @@ const TooltipContainer = styled(motion.div)<{
...
     $isWithHeader: boolean;
 }>`
-    background: ${({ theme }) => theme.backgroundNeutralBold};
-    color: ${({ theme }) => theme.textOnPrimary};
+    background: ${palette.darkGray300};
+    color: ${palette.lightWhiteAlpha1000};
     border-radius: ${borders.radii.sm};
     text-align: left;
-    box-shadow: ${({ theme }) => theme.boxShadowElevated};
+    border: solid 1.5px ${palette.darkGray100};
```
Comment (jvaclavik) [inline packages/components/src/components/Tooltip/Tooltip.tsx:43]: Yes it's correct. These colors shouldn't have light/dark mode

Comment (peter-sanderson) [reply packages/components/src/components/Tooltip/Tooltip.tsx:43]: Maybe lets add a comment explaing this into the code

File: packages/components/src/components/typography/TruncateWithTooltip/TruncateWithTooltip.tsx [hunk 2]
```diff
@@ -10,29 +10,46 @@ const EllipsisContainer = styled.div`
...
         });
         resizeObserver.observe(containerRef.current);
+        setIsResizeObserverReady(true);
 
         return () => resizeObserver.disconnect();
-    }, [children, containerRef]);
+    }, [children, scrollWidth, scrollHeight]);
+
+    useEffect(() => {
+        if (!isResizeObserverReady) return;
```
Comment (peter-sanderson) [inline packages/components/src/components/typography/TruncateWithTooltip/TruncateWithTooltip.tsx]: I think this does nothing and can be deleted.

**Review by peter-sanderson — APPROVED** [review]

**Comment by peter-sanderson** [comment]
Just please those 2 comments otherwise good.

**Pull request merged**